File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(int id, String license, Account driver, int passengers,
            Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(id, license, driver, passengers);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/uberVan.py
from car import Car


class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, passengers, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver, passengers)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
